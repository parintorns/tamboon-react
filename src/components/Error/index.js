import React from 'react'

export default function Spinner() {
  return <div>Error on loading data</div>
}
