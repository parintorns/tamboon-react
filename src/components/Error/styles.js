import styled from 'styled-components'

export const Wrapper = styled.div`
  position: absolute;
  top: 3px;
  padding: 5px 15px 5px 15px;
  border-radius: 3px;
  background-color: NavajoWhite;
`
