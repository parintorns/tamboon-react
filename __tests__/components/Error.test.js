import React from 'react'
import { shallow } from 'enzyme'

import Error from '../../src/components/Error'

let wrapper

const props = {}

beforeEach(() => {
  wrapper = shallow(<Error {...props} />)
})

describe('Render', () => {
  it('should render Spinner', () => {
    expect(wrapper.debug()).toMatchSnapshot()
  })
})
