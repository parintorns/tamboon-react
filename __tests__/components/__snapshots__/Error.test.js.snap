// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render should render Spinner 1`] = `
"<div>
  Error on loading data
</div>"
`;
